src/utils/data-utils.js

```diff
@@ -158,5 +158,6 @@ export function getAppAddressByName(apps, appName) {
 }
 
 export function getAppByName(apps, appName) {
-  return apps?.find(app => app.name === appName)
+  const regex = new RegExp(`^${appName}.*$`)
```

Comment:
could you remind me which are. the possible app names ?

Reply:
All the ones at `environment.js`

```
const DEFAULT_CONVICTION_APP_NAME = 'disputable-conviction-voting'
const DEFAULT_VOTING_APP_NAME = 'disputable-voting'
const DEFAULT_ISSUANCE_APP_NAME = 'dynamic-issuance'
const DEFAULT_AGREEMENT_APP_NAME = 'agreement'
```